### ISSUE TITLE
Set seed limiting properties of individual torrents

### ISSUE THREAD

**Issue by marcelpaulo** [rndusr/stig#60]
It might be really useful to be able to set these properties for individual torrents:

- seedIdleMode
- seedIdleLimit
- seedRatioMode
- seedRatioLimit

I guess these, plus `peer-limit`, are the only torrent properties which can't currently be set by stig. Before I discovered stig, I used transmission-remote, and I missed not being able to set `seedIdleMode` and `seedIdleLimit`.

My reason for needing to set these parameters is once again the files I send to my father using torrents: I configure global `idle-seeding-limit` to 12h, but my father sometimes takes longer to switch on his desktop (he's almost 80 years old and doesn't use his desktop all that frequently !), so I'd like to set `seedIdleLimit` to the torrents I add and seed for him to a value higher than the global value. Neither transmission's web interface nor transmission-remote offer a way of setting that property. I'd like to automate it through the torrent-done script: after I add one of these torrents just for seeding, the torrent-script done will be called, it would identify that it's a torrent I created, and would then set `seedIdleLimit` to unlimited. That is ... if stig allows to set that property.

Do you think that might fit and be useful within the toolset of stig ?

**Comment by rndusr** [comment]
Yes, that's something that stig should support. It's just not implemented yet.

BTW, you don't need to write an essay for every missing feature. If you miss a
feature, just ask for it. The fact that it is missing means I haven't gotten
around to implementing it yet, not that I don't want to.

If I say "You're crazy! Nobody will ever need that!" (I probably won't), you can
still convince me by explaining your use case.

On the other hand, it's definitely interesting to read about other people's
setups, so please continue to write essays if you like. :)

**Comment by marcelpaulo** [comment]
> BTW, you don't need to write an essay for every missing feature. If you miss a
> feature, just ask for it.

That's a deal ! I'll ask first, justify later, if necessary. I do tend to be long-winded, I must watch out. Thanks for pointing it out.
